pkg/tagger/types/types.go

```diff
@@ -23,4 +23,5 @@ type OriginInfo struct {
 	FromMsg       string
 	Cardinality   string
 	ProductOrigin ProductOrigin
+	OptOutEnabled bool
```

Comment:
should it be 
```suggestion
	OptOutEnabled *bool
```
in case the user forget to set it and because the default value is `true`